src/app/components/forms/user/password-reset/password-reset.component.html

```diff
@@ -0,0 +1,68 @@
+<h1>
+  {{
+    i18nService.i18nComponentsEnum.RESET_PASSWORD_COMPONENT_TITLE | translate
+  }}
+</h1>
+<h2>
+  {{
+    i18nService.i18nComponentsEnum.RESET_PASSWORD_COMPONENT_SUBTITLE | translate
+  }}
+</h2>
+
+<form
+  [formGroup]="passwordResetForm"
+  (ngSubmit)="onSubmit()"
+  autocomplete="off"
+  class="password-reset-form"
+>
+  <mat-form-field appearance="outline" class="full-width">
+    <mat-label>{{
+      i18nService.i18nFormsEnum.CURRENT_PASSWORD_LABEL | translate
+    }}</mat-label>
+    <input matInput formControlName="oldPassword" type="password" required />
+    <mat-error *ngIf="submitted() && oldPassword.hasError('required')">
+      {{ i18nService.I18nFormsEnum.CURRENT_PASSWORD_LABEL | translate }}
+    </mat-error>
+  </mat-form-field>
+
+  <mat-form-field appearance="outline" class="full-width">
```

Comment:
estas reglas que colocate de " class="full-width" "

Nosotros estamos usando bootstrap entonces en este caso puedes reemplazarlo por la regla que ya tenemos disponible. Quedaria como.


```suggestion
  <mat-form-field appearance="outline" class="w-100">
```

Reply:
Aplicaria hacer ese cambio en todos los lugares que estes usando esa regla de `full-width`